### DISCUSSION TITLE
mpd don't work with pipewire(system-wide)

### DISCUSSION THREAD

**Discussion by luffby** [MusicPlayerDaemon/MPD#1852] [Q&A]
Hello,
I'm using mpd as systemd service, and pipewire as user service.
In ncmpc track is forever paused.

So, journalctl -b | grep mpd

Aug 04 16:43:04 archpc systemd[1]: Listening on mpd.socket.
Aug 04 16:43:04 archpc mpd[545]: decoder: Decoder plugin 'wildmidi' is unavailable: configuration file does not exist: /etc/timidity/timidity.cfg
Aug 04 16:43:04 archpc mpd[545]: zeroconf: No global port, disabling zeroconf
Aug 04 16:44:55 archpc mpd[545]: exception: Failed to open "Pipewire Sound Server" (pipewire); Failed to connect stream: Host is down

mpd --verbose

config_file: loading file /home/trashy/.config/mpd/mpd.conf
exception: Default TCP listener setup failed, but this is okay because we have a $XDG_RUNTIME_DIR listener: Failed to bind to '[::]:6600'; Failed to bind socket: Address already in use
libsamplerate: libsamplerate converter 'Fastest Sinc Interpolator'
vorbis: Xiph.Org libVorbis 1.3.7
opus: libopus 1.4
sndfile: libsndfile-1.2.0
hybrid_dsd: The Hybrid DSD decoder is disabled because it was not explicitly enabled
decoder: Decoder plugin 'wildmidi' is unavailable: configuration file does not exist: /etc/timidity/timidity.cfg
simple_db: reading DB
input: Input plugin 'qobuz' is not configured: No Qobuz app_id configured
curl: version 8.1.2
curl: with OpenSSL/3.1.1

/etc/mpd.conf

music_directory "/home/music"
db_file "/var/lib/mpd/mpd.db"
playlist_directory "/var/lib/mpd/playlists"

audio_output {
       type "pipewire"
       name "Pipewire Sound Server"
}

Thanks in advance

**Comment by luffby** [comment]
ps -e -o ruser -o euser -o cmd | grep -E "mpd|pipewire|wireplumber" (if it helps somehow)

RUSER EUSER COMMAND
trashy   trashy   /usr/bin/pipewire
trashy   trashy   /usr/bin/wireplumber
mpd      mpd      /usr/bin/mpd --systemd

**Comment by MaxKellermann** [comment]
Is what you're trying to do even possible with PipeWire? This is a PipeWire-specific problem, not a MPD-specific problem. I can't help with PipeWire because I've never used it.
